client/footer.ejs

```diff
@@ -1,5 +1,12 @@
-<footer flex="none">
-  <a href="/about">About</a>
-  <a href="/tos">Terms</a>
-  <a href="/privacy-policy">Privacy</a>
+<footer layout="row">
+  <span flex class="footer-left">
+    <a href="/about">About</a>
+    <a href="/tos">Terms</a>
+    <a href="/privacy-policy">Privacy</a>
+  </span>
+  <span flex></span>
+  <span flex class="footer-right">
+    <a href="https://github.com/porygonco/porybox">GitHub</a> |
```

Comment:
The people who understand what a source code is, and are the most likely to contribute, will know what GitHub is.